examples/minimq-stm32h7/stm32h7-ethernet/Cargo.toml

```diff
@@ -20,7 +20,10 @@ cortex-m = "0.6.2"
 log = "0.4.8"
 
 [dependencies.smoltcp]
-version = "0.6.0"
+# Using specific git commit until PR 367 lands on crates.io
+# https://github.com/smoltcp-rs/smoltcp/pull/367
+git = "https://github.com/smoltcp-rs/smoltcp"
+rev = "bdfa442"
```

Comment:
The correct approach is to use [cargo patch](https://doc.rust-lang.org/edition-guide/rust-2018/cargo-and-crates-io/replacing-dependencies-with-patch.html) to avoid injecting this into all of the other toml files.

Reply:
I didn't know about cargo patch, that's a great feature!

Reply:
Yeah, I recently discovered it as well :)